src/player.js

```diff
@@ -320,8 +320,8 @@ export default class Player extends FakeEventTarget {
     this._textStyle = new TextStyle();
     this._createReadyPromise();
     this._createPlayerContainer();
-    this._appendPosterEl();
     this.configure(config);
+    (this.config.playback && !this.config.playback.autoplay) && this._appendPosterEl();
```

Comment:
This is the poster DOM placeholder, you want to avoid setting the poster source, not the entire DOM element

Reply:
@OrenMe you don't evenn want to avoid setting the poster source but only avoid to show it (i.e. posterManager.show())